instrumentation/apache-dubbo-2.7/library/src/main/java/io/opentelemetry/instrumentation/apachedubbo/v2_7/DubboTracer.java

```diff
@@ -46,6 +46,9 @@ public Context startClientSpan(String interfaceName, String methodName) {
   }
 
   public void end(Context context, Result result) {
+    if (result.hasException()) {
```

Comment:
If you do this, then you don't need to handle span status below.

Reply:
yes i got it:)